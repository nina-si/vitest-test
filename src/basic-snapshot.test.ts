import {describe, expect, it} from 'vitest'

describe('#sum', () => {
    it('toUpperCase', () => {
        const result = 'foobar'.toUpperCase();
        expect(result).toMatchSnapshot();
    });
})