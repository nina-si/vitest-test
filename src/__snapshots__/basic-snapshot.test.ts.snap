// Vitest Snapshot v1

exports[`#sum > toUpperCase 1`] = `"FOOBAR"`;
